maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -205,53 +196,65 @@ private DistroProperties resolveDistroProperties(Server server) throws MojoExecu
 
 		List<String> options = new ArrayList<>();
 
+		String currentDirectoryOption = null;
 		Distribution currentDirectoryDistribution = null;
 		File distroPropertiesFile = distroHelper.getDistroPropertiesFileFromDir();
 		if (distroPropertiesFile != null) {
 			try {
 				currentDirectoryDistribution = builder.buildFromFile(distroPropertiesFile);
 				if (currentDirectoryDistribution != null) {
-					options.add(currentDirectoryDistribution.getName() + " " + currentDirectoryDistribution.getVersion() + " from current directory");
+					currentDirectoryOption = currentDirectoryDistribution.getName() + " " + currentDirectoryDistribution.getVersion() + " from current directory";
+					options.add(currentDirectoryOption);
 				}
 			}
 			catch (Exception e) {
 				wizard.showWarning("Found " + distroPropertiesFile.getAbsolutePath() + " but unable to load this into a distribution:  " + e.getMessage());
 			}
 		}
 
-		options.add(O3_DISTRIBUTION);
-		options.add(O2_DISTRIBUTION);
+		options.add(REFAPP_2X_PROMPT);
+		options.add(REFAPP_3X_PROMPT);
 		options.add(PLATFORM);
-		String choice = wizard.promptForMissingValueWithOptions(SETUP_SERVERS_PROMPT, null, null, options);
 
+		String customDistroMessage = "Please specify distribution artifact";
+		String customDistroDefault = SDKConstants.REFERENCEAPPLICATION_2_4;
```

Comment:
By adding these to this top-level wizard step, we move the option to specify a custom distribution out from under the RefApp 2.x section and into the top-level section where it belongs.  This then allows us to focus the refapp 2.x section on paging through the published versions in Maven